cypress/integration/anton.sample/todoSample.page.js

```diff
@@ -0,0 +1,53 @@
+export class TodoSamplePage {
+
+    navigate () {
+        cy.visit('/')
+    }
+
+    todoHeader() {
+        return cy.get('h1')
+    }
+
+    newTodoInputField() {
+        return cy.get('.new-todo')
+    }
+
+    todoItem(itemText) {
+        return cy.contains('.todo-list li > div > label', itemText)
```

Comment:
Are you sure you need to build such complex css selector? Use more descriptive anchors, this path can be easily broken with new round of development.
In my case it was simple `.todo-list li`